plugins/modules/system/xfconf.py

```diff
@@ -264,6 +262,9 @@ def main():
         supports_check_mode=True
     )
 
+    # Force language to 'C' to ensure return values are always formatted in english, even in non-english environments
+    module.run_command_environ_update = dict(LANG='C')
```

Comment:
Hi @call-a3 , Thanks for the update!! I was going to stamp my approval in there, but the curiosity bug bit me: there are many locale-related variables and I never really took the time to understand how they relate to each other, so I wondered "why LANG? why not one of those LC_* ones"? Long story short, I found out that there is a priority order for those variables (from high to low): LANGUAGE, LC_ALL, LC_* (the others, specific ones like LC_TIME, LC_MESSAGES, etc), and then LANG. That came mostly from this neat and concise article about it: https://www.baeldung.com/linux/locale-environment-variables

So, we are setting LANG=C here, but if someone has any of the other higher-priority variables set, the module will fail. I am on a Linux Mint and out-of-the-box it sets LANGUAGE and LC_ALL with whatever was chosen during installation time.

Hence, I'd suggest using LANGUAGE instead of LANG, to prevent that situation:
```suggestion
    module.run_command_environ_update = dict(LANGUAGE='C')
```